agent-ndk/build.gradle

```diff
@@ -7,6 +7,26 @@ plugins {
     id "com.android.library"
     id "kotlin-android"
     id 'org.owasp.dependencycheck'
+    id 'jacoco'
+}
+
+task jacocoTestReport(type: JacocoReport, dependsOn: ['testDebugUnitTest', 'createDebugCoverageReport']) {
```

Comment:
This is the legacy task creation approach. Please convert to using Task.register() as shown [here](https://github.com/newrelic/newrelic-android-agent/blob/develop/instrumentation/build.gradle#L69-L79)